README.md

```diff
@@ -1,59 +1,62 @@
-#  <img src="docs/doctorkafka_logo.svg" alt="DoctorKafka logo" width="48"> &nbsp;&nbsp; Pinterest DoctorKafka
+#  <img src="docs/doctorkafka_logo.svg" alt="DoctorKafka logo" width="48px"> &nbsp;&nbsp; Pinterest DoctorKafka
 
-[![Build Status](https://travis-ci.org/pinterest/doctorkafka.svg)](https://travis-ci.org/pinterest/doctorkafka)
+***ATTENTION: `master` branch now points to DoctorKafka v3 (0.3.x). The legacy version (0.2.x) can be found under the `0.2.x` branch. 0.3.x and 0.2.x versions are INCOMPATIBLE.***
 
-DoctorKafka is a service for [Kafka] cluster auto healing and workload balancing.  DoctorKafka can automatically detect broker failure and reassign the workload on the failed nodes to other nodes. DoctorKafka can also perform load balancing based on topic partitions's network usage, and makes sure that broker network usage does not exceed the defined settings. DoctorKafka sends out alerts when it is not confident on taking actions.
+DoctorKafka is a service for [Kafka] cluster auto-healing and workload balancing.  DoctorKafka can automatically detect broker failure and reassign the workload on the failed nodes to other nodes. DoctorKafka can also perform load balancing based on topic partitions's network usage, and makes sure that broker network usage does not exceed the defined settings. DoctorKafka sends out alerts when it is not confident on taking actions.
 
-#### Features   
+In DoctorKafka v3, a plugin framework is introduced to provide greater flexibility to add more features. The functionality of legacy versions are perserved in the default plugins provided.
```

Comment:
change `v3` to `0.3` ?    

`greater flexibility` --> `flexibility in collecting kafka cluster status data and taking customized actions` ?